Add FAQ (#54)
Fabulously-Optimized/wiki: en-us/faq.md

> # FAQ

Reviewer: "Vanilla launcher FAQ"

> 

Reviewer: add a short subtext about what this FAQ is about

### How to install?

> - There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions) to install FO on the launcher that you use.

Reviewer: no no, this FAQ is about vanilla specifically
so link to vanilla instructions

### So I just copy the mods, right?

> - Well no because each FO instance / ZIP contains configs and a custom resource pack that is built to help the user navigate through Mod Menu. If you only copy the mods over, the configs or overrides would not apply and you would configure the mods yourself which could lead to not having the right settings to replicate OptiFine.

Reviewer: revise the phrasing a bit
something along the lines of 
"No. Fabulously Optimized consists of mods, configs and a resource pack. If you only..."

### What is the MultiMC ZIP?

> - MultiMC ZIP is packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge Modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.

Reviewer: ```suggestion
- MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge Modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
```

> - MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge Modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.

Reviewer 2: ```suggestion
- The MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.
```

### What is in the ZIP file?

> - In both the CurseForge and MultiMC ZIP files, they contain configs for each mod and resource packs which are essential for FO to run. Without these files, the modpack would not be configured properly which could lead to not having the right settings to replicate OptiFine.

Reviewer: this should be an hierarchical overview of each folder and important file, however you do not have to mention individual config files (so just combine them under "mod configs" or so)

example:

* folder1 - description
* folder2 - description
   * subfolder1 - description

### Why try a different launcher?

> - There are many launchers out there that have different features to one another. For example, if you want a launcher that you cna easily update the pack on, we recommend either using GDLauncher or the CurseForge Launcher. But if you want a launcher that can be used offline with the modpack, we recommend using MultiMC and GDLauncher.

Reviewer: the point is: why not use vanilla launcher; what are the benefits of using alternative launchers
word it as such

### Why is FO not a client?

> - FO is not a client because simply we would need to be able to code a client which would require people to know the programming language that we are writing the client in. Also, we can't bundle a bunch of mods without thinking about the license that they are using. Some licenses don't allow for distribution or for commercial use which could affect some mods being excluded from the client.

Reviewer: focus more on the flexibility part, making it easy for the user and creator to switch mods at will

### Is there an easier way to install FO?

> - For now, we recommend using other launchers like CurseForge Launcher, MultiMC and GDLauncher as provide an easy way to install the modpack as well as to upgrade it. However, if you want to install FO on the Vanilla Launcher, you can follow this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) until we have a better solution. Soon, we are going to release a Vanilla Installer which provides an easier way to install the modpack on the Vanilla Launcher.

Reviewer: focus more on like the installer is coming, but the user will always get a better experience on alternative launchers

> - For now, we recommend using other launchers like CurseForge Launcher, MultiMC and GDLauncher as provide an easy way to install the modpack as well as to upgrade it. However, if you want to install FO on the Vanilla Launcher, you can follow this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) until we have a better solution. Soon, we are going to release a Vanilla Installer which provides an easier way to install the modpack on the Vanilla Launcher.

Reviewer: Proofread and revise wording on the whole thing with QuillBot
Make sure it is written from vanilla launcher, not generic perspective

> - For now, we recommend using other launchers like CurseForge Launcher, MultiMC and GDLauncher as provide an easy way to install the modpack as well as to upgrade it. However, if you want to install FO on the Vanilla Launcher, you can follow this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) until we have a better solution. Soon, we are going to release a Vanilla Installer which provides an easier way to install the modpack on the Vanilla Launcher.

Reviewer 2: > Soon, we are going to release a Vanilla Installer which provides an easier way to install the modpack on the Vanilla Launcher.

again with the capitalization, maybe link to the repo (https://github.com/Fabulously-Optimized/vanilla-installer)
Hoping we can get a beta out soon-ish, but not sure yet.

# Vanilla launcher FAQ

> This FAQ gives answers to questions you might have about the vanilla launcher. Please join our [Discord](https://discord.gg/yxaXtaQqdB) and visit the 'support' channel if you have any additional questions. 

Reviewer: ```suggestion
This FAQ gives answers to questions you might have about using Fabulously Optimized with the vanilla launcher. Please join our [Discord](https://discord.gg/yxaXtaQqdB) and visit the 'support' channel if you have any additional questions. 
```

### How to install?

> - To install FO on the Vanilla Launcher, please follow this guide [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla)

Reviewer: ```suggestion
- To install FO on the Vanilla Launcher, [please follow this guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla).
```

### So I just copy the mods, right?

> - No. Fabulously Optimized consists of mods, configs and resourcepacks. The settings would not be applied if you simply copied the mods over, and because you would have to setup the mods manually, you might not have the right settings to precisely match OptiFine.

Reviewer: Last sentence could still be worded better.
```suggestion
- No. Fabulously Optimized consists of mods, configs and resourcepacks. If you skip configurations, there is a high chance of some resource packs not working at all, mods may have unwanted features enabled or wanted features disabled. If you skip resource packs, you'll lose out on the descriptions that explain what each mod does in Mod Menu.
```

### Why try a different launcher?

- There are a variety of launchers available, each with unique features. If you use the vanilla launcher, you miss out on many features, including the ability to update the modpack and the version of Minecraft, add any mod with a few simple clicks, and finally download the modpack more easily.

### Why is FO not a client?

- FO is also not a client because simply we would need explicit permission from the mod authors to use their mods in a client and the authors of each mod would receive no benefits by being included in a client. We would also need to be able to code a client which would require lots of time and maintenance to keep the client up to date with the latest version of Minecraft. By being a modpack, this makes it easier for the creator and user to add their own mods in and use Fabulously Optimized as base pack for their own mods they want to add in. This can also make it easier to update the modpack by adding in replacement mods while waiting for other mods in the pack to update.

### Is there an easier way to install FO?

> - Soon, there will be a vanilla installer that will make it easier to install the modpack on the vanilla launcher. You can check out the [Github Repo](https://github.com/Fabulously-Optimized/vanilla-installer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for the installer to see when a beta could be released. We advise using other launchers like CurseForge Launcher, MultiMC, and GDLauncher to make it simple and easy to install and update the modpack for the time being until the vanilla installer is released.

Reviewer 2: ```suggestion
- Soon, there will be a program that will make it easier to install the modpack on the vanilla launcher. You can check out the [GitHub Repo](https://github.com/Fabulously-Optimized/vanilla-installer/releases) for the installer to see when a beta could be released. We advise using other launchers like CurseForge Launcher, MultiMC, and GDLauncher to make it simple and easy to install and update the modpack for the time being until the vanilla installer is released.
```
We probably won't announce in an issue when a beta of VI has been released, so link to the releases page. Otherwise just of couple spelling fixes :P

> - Soon, there will be a vanilla installer that will make it easier to install the modpack on the vanilla launcher. You can check out the [Github Repo](https://github.com/Fabulously-Optimized/vanilla-installer/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) for the installer to see when a beta could be released. We advise using other launchers like CurseForge Launcher, MultiMC, and GDLauncher to make it simple and easy to install and update the modpack for the time being until the vanilla installer is released.

Reviewer: ```suggestion
- Soon, there will be an installer that will make it easier to install the modpack on the vanilla launcher. You can check out the [relevant issue](https://github.com/Fabulously-Optimized/fabulously-optimized/issues/110) to see when a beta could be released. However, we still advise you to prefer other launchers like CurseForge Launcher, as they make installing, updating and adding mods much easier than vanilla ever can, and support the development of the modpack alongside it.
```

# Vanilla launcher FAQ

> ## What is this?

Reviewer: ```suggestion

```
pretext does not need a header

> - This FAQ gives answers to questions you might have about the vanilla launcher. Please join our [Discord](https://discord.gg/yxaXtaQqdB) and visit the 'support' channel if you have any additional questions. 

Reviewer: not resolved

> - This FAQ gives answers to questions you might have about the vanilla launcher. Please join our [Discord](https://discord.gg/yxaXtaQqdB) and visit the 'support' channel if you have any additional questions. 

Reviewer: yes, remove them

### How to install?

- There are many ways to install FO on multiple launchers. Please use this [guide](https://fabulously-optimized.gitbook.io/modpack/readme/install-instructions#minecraft-launcher-vanilla) to install FO on the launcher that you use.

### So I just copy the mods, right?

- No. Fabulously Optimized consists of mods, configs and a resource pack that is built to help the user naviagte through Mod Menu. If you simply copy the mods over, the settings would not be applied, and since you would have to configure the mods yourself, you might not have the proper settings to accurately replicate OptiFine.

### What is the MultiMC ZIP?

- The MultiMC ZIP is a packaged instance for the launcher called MultiMC. It is distributed in CurseForge because MultiMC itself no longer supports downloading CurseForge modpacks and downloads from the ZIP file support the creator of FO. This also allows for a better installation of FO for vanilla launcher users.

### What is in the ZIP file?

> - In both the CurseForge and MultiMC ZIP files, they contain configs for each mod and resource packs which are essential for FO to run. The config folder contain configs for each mod in FO so the modpack is configured perfectly when you first start up the modpack, the mods folder contains mods that are essential to replicate OptiFine's features, and the resource folder contains a resource pack that helps the user navigate through Mod Menu. There is also a file called 'Copy all 3 folders!', this file makes it easier to update the pack rather than having to download and make a new instance each time there is a new update.

Reviewer: - folder
   - tree
        - please
- refer to this comment again: https://github.com/Fabulously-Optimized/wiki/pull/54#discussion_r1005459197

Reviewer: not resolved

Reviewer: It is still a paragraph but I'd prefer a file tree with bullet points here, to describe each file individually

Reviewer: Yes, probably separate the name and description with a dash.

### Why try a different launcher?

> - There are a variety of launchers available, each with unique features. If you use the vanilla launcher, you miss out on many features, including the ability to update the modpack and the version of Minecraft, add any mod with a few simple clicks, and finally download the modpack more easily.

Reviewer: reorder the sentences and sentence contents by priority

priority:

1. easy install
2. easy update
3. adding mods
4. unique features

### Why is FO not a client?

> - FO is not a client because simply we would need to be able to code a client which would require people to know the programming language that we are writing the client in. By being a modpack, this makes it easier for the creator and user to add their own mods in and use Fabulously Optimized as base pack for their own mods they want to add in. This can also make it easier to update the modpack by adding in replacement mods while waiting for other mods in the pack to update.

Reviewer: incorporate more things from here:
https://github.com/Fabulously-Optimized/fabulously-optimized/issues/101#issuecomment-888805246

programming itself is not the highest concern, rather licensing and integration are
so reword it based on priority and some extra points from the link

### Why try a different launcher?

> - There are a variety of launchers available, each with unique features. If you use the Vanilla Launcher, you miss out on many features, including the ability to update the modpack and the version of Minecraft, add any mods with a few simple clicks, and finally download the modpack more easily.

Reviewer 2: Should `Vanilla Launcher` really be capitalized? It's not the actual name of the program (that's Minecraft Launcher, but it doesn't make any sense here), so it shouldn't be capitalized like it imo
